README.md

```diff
@@ -1,98 +1,81 @@
-VOLTTRON™ is an open source platform for distributed sensing and control. The platform provides services for collecting and storing data from buildings and devices and provides an environment for developing applications which interact with that data.
+This package provides web services for the VOLTTRON™ platform.
+This includes a RESTful API for interacting with the platform
+and utility pages for administration and certificate management.
+This library cannot be installed as a VOLTTRON agent as is.
+Rather, it must be installed as a python package.
+
+VOLTTRON™ is an open source platform for distributed sensing and control.
+The platform provides services for collecting and storing data from buildings
+and devices and provides an environment for developing applications which
+interact with that data.
 
 [![Run Pytests](https://github.com/eclipse-volttron/volttron-lib-web/actions/workflows/run-test.yml/badge.svg)](https://github.com/eclipse-volttron/volttron-lib-web/actions/workflows/run-test.yml)
 [![pypi version](https://img.shields.io/pypi/v/volttron.svg)](https://pypi.org/project/volttron-core/)
 
+## Requirements
+* python >=3.8
+
 ## Installation with Web
+This library can be installed using pip:
 
 ```bash
 > pip install volttron-lib-web
 ```
 
-### Quick Start
-
- 1. Start the platform
-    ```bash
-    > volttron -vv -l volttron.log &>/dev/null &
-    ```
-
- 2. Install listener agent
-    ```bash
-    > vctl install volttron-listener
-    ```
-
- 3. View status of platform
-    ```bash
-    > vctl status
-    ```
-
- 4. Shutdown the platform
-    ```bash
-    > vctl shutdown --platform
-    ```
-
-Full VOLTTRON documentation available at [VOLTTRON Readthedocs](https://volttron.readthedocs.io)
-
-## Contributing to VOLTTRON
-
-Please see the [contributing.md](CONTRIBUTING.md) document before contributing to this repository.
-
-## Development of VOLTTRON
-
-### Environment
-
-VOLTTRON uses [Poetry](https://python-poetry.org/), a dependency management and packaging tool for Python. If you don't have Poetry installed on your machine, follow [these steps](https://python-poetry.org/docs/#installation) to install it on your machine.
+If VOLTTRON was not already installed, it will be pulled in as a dependency
```

Comment:
This library depends on volttron...right.

Reply:
Removed.